crates/sui-single-node-benchmark/src/benchmark_context.rs

```diff
@@ -32,10 +32,14 @@ impl BenchmarkContext {
         workload: Workload,
         benchmark_component: Component,
         checkpoint_size: usize,
+        print_sample_tx: bool,
```

Comment:
not sure what `print_sample_tx` this means here as opposed to where we `info` the txn